README.md

```diff
@@ -107,7 +110,7 @@ make chart-deploy VALUESFILE=yourvaluesfile.yaml
 
 ### Verify the Installation
 
-Verify the Gateway Sidecar Container is running by describing the gateway pod. 
+Verify the Gateway Sidecar Container is running by checking the status of gateway pod belonging to the `kubeslice-system` namespace.
```

Comment:
Only here, we have used `Gateway Sidecar` in title case. 

Can we use the same title case for `gateway sidecar` above in the running paragraphs (wherever applicable)?